Automattic/woocommerce-payments
# New account period notice in Simplify Deposits UI

Fixes https://github.com/Automattic/woocommerce-payments/issues/5727

#### Changes proposed in this Pull Request

When account has not completed the [new account waiting period](https://woocommerce.com/document/woocommerce-payments/deposits/deposit-schedule/#section-1), show a notice: Your first deposit is held for seven business days. [Why?](https://woocommerce.com/document/woocommerce-payments/deposits/deposit-schedule/#section-1).

#### Testing instructions

* Enable the feature flag `_wcpay_feature_simplify_deposits_ui`.
* To be in the new account waiting period, either onboard a new account or change your local code [here](https://github.com/Automattic/woocommerce-payments/blob/53938f33b37307f469ec9a947de9891115f8b4e9/includes/admin/class-wc-payments-admin.php#L487) by adding

```
$account_status_data['deposits']['completed_waiting_period'] = false;
```

* Confirm that the notice shows up.
* Test on mobile screen as well. It should look like this.

![Screenshot 2023-03-29 at 01 12 59](https://user-images.githubusercontent.com/73803630/228331246-0293478e-7b02-48c5-88e7-4a667d48a751.png)


-------------------


**Post merge**
